CMakeLists.txt

```
@@ -565,6 +568,21 @@ ELSE()
 	SET (STATUS_DEBUG_FPE "Disabled")
 ENDIF(WANT_DEBUG_FPE)
 
+if(WANT_DEBUG_ASAN)
+	add_compile_options(-fsanitize=address)
+	add_link_options(-fsanitize=address)
```

ASAN on MSVC works when added to the compiler but it doesn't support linking. Will lead to an error. You don't need the linker flags on msvc.
```suggestion
    if(NOT MSVC)
	    add_link_options(-fsanitize=address)
	endif()
```

---

Better to support MSVC instead.

---

We ran into some other issues with MSVC on Discord, so I'm not sure if I should still apply this suggesstion?

---

Feel free to discard. My issues were unrelated to the linker flag.